README.md

```diff
@@ -135,9 +136,12 @@ See [make-defaults.mk](make-defaults.mk) for a full list of manually configurabl
 
 Official releases of DC/OS can be found at <http://dcos.io/releases/>.
 
-By default, DC/OS Docker (`./configure`) downloads the latest **stable** version of DC/OS.
+By default, `./configure --auto` downloads the latest **stable** version of DC/OS.
```

Comment:
technically, even the non-auto version also downloads the latest stable if you just hit enter through all the prompts (which is what --auto does)